query/src/sql/planner/binder/project.rs

```diff
@@ -74,7 +74,7 @@ impl<'a> Binder {
     pub(super) async fn normalize_select_list(
         &mut self,
         select_list: &[SelectTarget<'a>],
-        input_context: &BindContext,
+        input_context: &mut BindContext,
```

Comment:
This `mut` can be removed.

Reply:
Make the `BindContext` as the first parameter of the function is more make sense.

Reply:
> Make the `BindContext` as the first parameter of the function is more make sense.

I've just opened a pr for this https://github.com/datafuselabs/databend/pull/5371.

Reply:
I see it, hope they won't conflict :D